src/test-setup.js

```diff
@@ -79,6 +79,22 @@ global.OCP = {
 }
 global.IS_DESKTOP = false
 
+/**
+ * Polyfill for Blob.prototype.arrayBuffer
+ */
```

Comment:
For me, it is a bit not obvious why we need to polyfill something that already exists in Node.js [since v15](https://developer.mozilla.org/en-US/docs/Web/API/Blob/arrayBuffer#browser_compatibility) and is web standard compatible [since v18](https://nodejs.org/de/blog/announcements/v18-release-announce#other-global-apis).

```suggestion
/**
 * Polyfill for Blob.prototype.arrayBuffer
 * Required as jsdom breaks Nodejs's native Blob
 * @see: https://github.com/jsdom/jsdom/issues/2555
 */
```